nifi-nar-bundles/nifi-grpc-bundle/nifi-grpc-nar/src/main/resources/NOTICE

```diff
@@ -0,0 +1,293 @@
+nifi-grpc-nar
+Copyright 2017 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
```

Comment:
I believe the assembly NOTICE needs to include the bundled dependency NOTICE info also (although it looks like there are some already in there from this list)